Cargo.toml

```
@@ -26,7 +26,7 @@ enr = { version = "0.5", default-features = false, features = ["rust-secp256k1"]
 enum-primitive-derive = "0.2"
 ethereum = { version = "0.7", default-features = false }
 ethereum-forkid = "0.5"
-ethereum-interfaces = { git = "https://github.com/ledgerwatch/interfaces", features = ["sentry"] }
+ethereum-interfaces = { git = "https://github.com/ledgerwatch/interfaces", rev = "b746fe2", features = ["sentry"] }
```

```suggestion
ethereum-interfaces = { git = "https://github.com/ledgerwatch/interfaces", features = ["sentry"] }
```

We have Cargo.lock for this